### DISCUSSION TITLE
BUG: User signatures reset to blank

### DISCUSSION THREAD

**Discussion by gitcodehelp** [sheltermanager/asm3#1481] [Q&A / Help] [answered]
To reproduce:
1) User menu > Change User Settings > (draw a signature with mouse) > Save
2) User menu > Change User Settings > (do nothing, or change a setting) > Save
3) User menu > Change User Settings > (the signature is now blank)

ASM v48 (release version)

**Comment by bobintetley** [comment, marked as answer]
Thanks for the bug report. I've applied a fix to the codebase now, commit 6798a48d7c19bf5e7d747b302f1c8aed4836177b

**Reply by gitcodehelp** [reply]
Thank you!
